madgraph/interface/common_run_interface.py

```diff
@@ -7587,7 +7587,7 @@ def open_file(self, answer):
                 answer = 'plot'
             else:
                 answer = self.cards[int(answer)-self.integer_bias]
-
+        path = ''
```

Comment:
this is unrelated to the issue at hand, right?

Reply:
yes it is uncorrelated, this fixed another issue in another branch and made a mistake to include this in this PR (but would prefer to keep it anyway)